### PR TITLE
Fix labeler action trigger

### LINKED ISSUE #492: Labeler action don't work on fork repository
## Overview
Labeler action don't work on fork repository.
Probably, it is because the trigger is pull_request.

## Reference
https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

### PR DESCRIPTION
**Pull request by shin-usu**
## Issue
- close #492 

## Overview (Required)
- Fix labeler action trigger

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: 'PR Labeler'
 
 on:
-  pull_request:
+  pull_request_target:
```
Comment (takahirom) [inline .github/workflows/labeler.yml:4]: When using pull_request_target, it's advisable to avoid using actions/checkout@v4 due to security considerations.

Comment (shin-usu) [reply .github/workflows/labeler.yml:4]: Oh, really? Thanks for your information.
Would it be better to lower version of the checkout action?

Comment (takahirom) [reply .github/workflows/labeler.yml:4]: I would recommend separating the tasks of saving the PR number and adding labels to the PR. You can find many examples in this repository.
https://github.com/DroidKaigi/conference-app-2024/blob/main/.github/workflows/save-pr-number-for-removing-waiting-label.yml
https://github.com/DroidKaigi/conference-app-2024/blob/main/.github/workflows/remove-waiting-label.yml

https://github.com/DroidKaigi/conference-app-2024/blob/main/.github/workflows/detekt.yml
https://github.com/DroidKaigi/conference-app-2024/blob/main/.github/workflows/detekt-comment.yml

https://github.com/DroidKaigi/conference-app-2024/blob/main/.github/workflows/screenshot-comparison.yml
https://github.com/DroidKaigi/conference-app-2024/blob/main/.github/workflows/screenshot-comparison-comment.yml

Comment (takahirom) [reply .github/workflows/labeler.yml:4]: You can see the blog about this.
https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Comment (shin-usu) [reply .github/workflows/labeler.yml:4]: OK 👍🏻
I will fix and recreate pull request.

**Pull request closed without merging**